src/scancode/extract_cli.py

```diff
@@ -90,11 +90,12 @@ class ExtractCommand(utils.BaseCommand):
 @click.option('--quiet', is_flag=True, default=False, help='Do not print any summary or progress message.')
 @click.option('--shallow', is_flag=True, default=False, help='Do not extract recursively nested archives (e.g. not archives in archives).')
 @click.option('--replace-originals', is_flag=True, default=False, help='Replace extracted archives by the extracted content.')
+@click.option('--ignore', default=[], multiple=True, help='Ignore files with a particular extension')
```

Comment:
`--ignore` should work the same way as it does in scancode, meaning you can ignore whole filename or directories, not just file extensions.